Mailer/MailerInterface.php

```diff
@@ -31,4 +31,14 @@ public function sendConfirmationEmailMessage(UserInterface $user);
      * @param UserInterface $user
      */
     public function sendResettingEmailMessage(UserInterface $user);
+
+    /**
+     * Send an email to a user to confirm the changed email address.
+     *
+     * @param UserInterface $user
+     * @param string $confirmationUrl
+     * @param string $toEmail
+     */
+    public function sendUpdateEmailConfirmation(UserInterface $user, $confirmationUrl, $toEmail);
```

Comment:
Changing the interface is a BC break.

Reply:
indeed. That slipped my attention, sorry.

It is an easy one to fix though. And only users that use their own mailer that does not extend the FosUserBundle mailer need to do anything.